app/auth-portal/src/pages/WorkspaceAuthTokensPage.vue

```diff
@@ -13,103 +13,111 @@
+              Generate Automation Token
             </VButton>
           </div>
-          <Stack
-            v-if="createAuthToken.state.value"
-            class="border-2 outline-2 rounded p-8"
-          >
-            <div>Token successfully created!</div>
-            <div class="text-lg font-bold">
-              This is the last time you will ever see this token value.
-            </div>
-            <VormInput
-              :modelValue="createAuthToken.state.value"
-              type="textarea"
-              label="Your API Token"
-              disabled
-            />
-            <div>
-              Click the button below to copy it before navigating away or doing
-              anything else!
-            </div>
-            <VButton
-              icon="clipboard-copy"
-              tone="action"
-              label="Copy API Token To Clipboard"
-              class="mt-xs"
-              clickSuccess
-              successText="Copied to clipboard!"
-              @click="copyToken"
-            />
-          </Stack>
           <ErrorMessage :asyncState="authTokens" />
           <div v-if="authTokens.state.value" class="relative">
             <Stack>
-              <table
-                class="w-full divide-y divide-neutral-400 dark:divide-neutral-600 border-b border-neutral-400 dark:border-neutral-600"
-              >
+              <table class="w-full">
                 <thead>
                   <tr
-                    class="children:pb-xs children:px-md children:font-bold text-left text-xs uppercase"
+                    class="children:py-xs children:px-sm children:font-bold text-left text-xs uppercase bg-black"
                   >
                     <th scope="col">Name</th>
                     <th scope="col">Created</th>
                     <th scope="col">Expires</th>
-                    <th scope="col">Revoke</th>
+                    <th class="text-center" scope="col">Revoke</th>
                   </tr>
                 </thead>
                 <tbody
                   class="divide-y divide-neutral-300 dark:divide-neutral-700"
                 >
                   <AuthTokenListItem
-                    v-for="authToken of Object.values(
-                      authTokens.state.value,
-                    ).reverse()"
+                    v-for="authToken of listedTokens"
                     :key="authToken.id"
                     :authToken="authToken"
                     :workspace="workspace"
-                    @revoked="delete authTokens.state.value[authToken.id]"
                     @renamed="(newName) => (authToken.name = newName)"
+                    @revoked="delete authTokens.state.value[authToken.id]"
                   />
                 </tbody>
               </table>
             </Stack>
           </div>
         </Stack>
       </div>
+      <Modal ref="tokenDisplayModalRef" size="lg" title="Token Generated">
+        <ErrorMessage
```

Comment:
Nice touch making it red until you copy!

... though is there a `SuccessMessage` or something one could use for the happy case?

Reply:
I think the move here would be to have make this into a statusMessage or messageBox component eventually, since we don't have anything else